lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -2,7 +2,7 @@
 from flatten_json import flatten
 
 
-def lusid_response_to_data_frame(lusid_response, rename_properties: bool = False):
+def lusid_response_to_data_frame(lusid_response, rename_properties: bool = False, rename_mapping: dict = None):
```

Comment:
maybe rename to something `column_names`

Reply:
Sure, renamed to "column_name_mapping"